cdap-proto/src/main/java/co/cask/cdap/proto/StreamViewSpecification.java

```diff
@@ -0,0 +1,57 @@
+/*
+ * Copyright © 2015 Cask Data, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not
+ * use this file except in compliance with the License. You may obtain a copy of
+ * the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations under
+ * the License.
+ */
+package co.cask.cdap.proto;
+
+import co.cask.cdap.api.data.format.FormatSpecification;
+
+import java.util.Objects;
+
+/**
+ * Represents the configuration of a stream view, used when creating or updating a view.
+ */
+public class StreamViewSpecification {
+
+  private final FormatSpecification format;
```

Comment:
Not sure we can both add view name to `ViewSpecification` and use `ViewSpecification` instead of `FormatSpecification`, since we still assume there's a "default" `FormatSpecification` per stream. The config is in `FormatSpecification`.